frontend/src/components/Columns.jsx

```diff
@@ -19,22 +19,22 @@ function Columns() {
       setDragging(true);
     }, 0);
   };
+
   const handleDragEnter = (e, targetItem) => {
-    console.log('Drop target: ', targetItem);
     if (dragItemNode.current !== e.target) {
-      console.log('old: ', list);
+      const targetGroupIndex = targetItem.groupIndex;
```

Comment:
https://wesbos.com/destructuring-renaming

```suggestion
      const { groupIndex : targetGroupIndex } = targetItem;
```

Reply:
https://github.com/hegedus-adam/kanban-board/pull/15/commits/af2c91558cf52c265e7a384e53b9bf158efc936b